smoke_tests/request_and_confirm_test.rb

```diff
@@ -0,0 +1,132 @@
+require 'capybara/poltergeist'
+
+class SmokeTest
+  include Capybara::DSL
+
+  PRISON_EMAIL = 'pvb.cardiff@maildrop.dsd.io'
+  VISITOR_EMAIL = 'test@digital.justice.gov.uk'
+
+  def run
+    with_custom_config
+    run_internal
+  rescue StandardError => e
+    pp e
+    pp e.backtrace
+    save_screenshot('error.png', full: true)
+  end
+
+  private
+  def run_internal
+    visit '/prisoner'
+    enter_prisoner_information
+    continue
+    
+    raise "Can't find Visitor 1" unless page.has_content?('Visitor 1')
```

Comment:
Oh, and if you want to reproduce what I'm doing, I've created a job in Jenkins. We don't have the new environment variables on the host-under-test yet, but it does at least appear to be hitting the app.